mesonbuild/ast/printer.py

```diff
@@ -57,7 +59,7 @@ def visit_NumberNode(self, node: mparser.NumberNode) -> None:
         node.lineno = self.curr_line or node.lineno
 
     def escape(self, val: str) -> str:
-        return val.replace('\\', '\\\\').replace("'", "\'")
+        return val.translate(self.escape_trans)
```

Comment:
Commit message seems unrelated to this change.

Reply:
> Commit message seems unrelated to this change.

You're right, I changed the commit message